pkg/ebpf/c/tracee.bpf.c

```diff
@@ -4781,6 +4781,9 @@ int BPF_KPROBE(trace_security_file_mprotect)
     unsigned long reqprot = PT_REGS_PARM2(ctx);
 
     if (should_submit(SECURITY_FILE_MPROTECT)) {
+        if (!should_trace(&data.context))
+            return 0;
```

Comment:
I would place this just after the init_event_data and not here, so it would catch all cases

Reply:
Ok, makes sense.
Soon I plan to add caching for should_trace decisions, so that won't be a problem anymore.